src/motor-interface-pru.p

```diff
@@ -174,6 +185,9 @@ INIT:
 	SBCO r0, C4, 4, 4
 
 	MOV r2, 0		; Queue address in PRU memory
+	MOV r28, 0		; Status register in PRU memory,
+	 				; top byte for current queue position,
```

Comment:
Can you align the ';' with the line before ?
(or does it just look off in the diff because of some TAB ?)